eth_portfolio/structs.py

```diff
@@ -118,11 +118,16 @@ def __init_subclass__(cls, **kwargs):
         # Use the class name exactly as it is defined (e.g., TokenTransfer)
         cls_name = cls.__name__
 
+        # Replace {cls_name} in the class-level docstring
```

Comment:
why are we re-adding this? It doesn't do anything in our use case

Reply:
I think it means the attribute docstrings which are technically a part of the overall class docstrings

Reply:
but we're doing that immediately below